doc/source/whatsnew/v0.25.0.rst

```diff
@@ -18,10 +18,7 @@ including other versions of pandas.
 
 Other Enhancements
 ^^^^^^^^^^^^^^^^^^
-
--
--
--
+:func:`DataFrame.plot` allows to expose symlog scaling for axis.
```

Comment:
can you make this more informative, something like 
DataFrame.plot keyword ``logy`` and ``logx`` now accept the value ``sym`` to .....

Reply:
add the issue number (or if not issue, this PR number)

Reply:
thanks for the review, changed! @jreback